connection.go

```diff
@@ -88,6 +89,9 @@ func (sc *snowflakeConn) exec(
 	headers["Content-Type"] = headerContentTypeApplicationJSON
 	headers["accept"] = headerAcceptTypeApplicationSnowflake // TODO v1.1: change to JSON in case of PUT/GET
 	headers["User-Agent"] = userAgent
+	if serviceName, ok := sc.cfg.Params[serviceName]; ok {
```

Comment:
the sc.cfg.Params[] would be updated by populateSessionParameters() function, called by driver.Open() and connection.exec()